### PR TITLE
fix: remove redundant custom manifests copying

### PR DESCRIPTION
**Pull request by maxisam**
# Description

#285 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Small minor change not affecting the Ansible Role code (GitHub Actions Workflow, Documentation etc.)

### CODE REVIEW

File: tasks/rke2.yml [hunk 1]
```diff
@@ -231,16 +231,6 @@
...
-- name: Copy Custom Manifests
-  ansible.builtin.template:
-    src: "{{ item }}"
-    dest: "{{ rke2_data_path }}/server/manifests/"
-    owner: root
-    group: root
-    mode: 0644
-  with_items: "{{ rke2_custom_manifests }}"
-  when: rke2_custom_manifests
-
```
Comment (MonolithProjects) [inline tasks/rke2.yml:243]: I think we should keep this Task and rather replace the following Block with

```yaml
- name: Copy Static Pods
  ansible.builtin.copy:
    src: "{{ item }}"
    dest: "{{ rke2_data_path }}/agent/pod-manifests/"
    owner: root
    group: root
    mode: 0644
  with_items: "{{ rke2_static_pods }}"
  when: rke2_static_pods
```
 to keep the original behavior.
Basically we need to revert this change https://github.com/lablabs/ansible-role-rke2/pull/260/files#diff-18777598e84dcc9ba07f83aaee64bfcdf001257e84ed46f349bd1a84a178dbebR239-R266

**Review by MonolithProjects — CHANGES_REQUESTED** [review]

File: tasks/rke2.yml (hunk 1 above)
Comment (maxisam) [reply tasks/rke2.yml:243]: Either way is fine with me. Feel free to close it.
